e2e-tests/gef/cypress/e2e/clone/facility-end-date-enabled/clone-v1-to-v1.spec.js

```diff
@@ -0,0 +1,138 @@
+import relative from '../../relativeURL';
+import { BANK1_MAKER1 } from '../../../../../e2e-fixtures/portal-users.fixture';
+import cloneGEFDeal from '../../pages/clone-deal';
+import mandatoryCriteria from '../../pages/mandatory-criteria';
+import nameApplication from '../../pages/name-application';
+import applicationDetails from '../../pages/application-details';
+import { submitButton } from '../../partials';
+import aboutFacilityUnissued from '../../pages/unissued-facilities-about-facility';
+import facilityEndDate from '../../pages/facility-end-date';
+import bankReviewDate from '../../pages/bank-review-date';
+import { MOCK_APPLICATION_AIN } from '../../../fixtures/mocks/mock-deals';
+import { anUnissuedCashFacility, anIssuedCashFacility } from '../../../../../e2e-fixtures/mock-gef-facilities';
+import { DEAL_STATUS, FACILITY_TYPE } from '../../../fixtures/constants';
+
+context('Clone version 1 deal to version 1', () => {
```

Comment:
How/why is it possible to clone a version 1 deal to the same version?

Reply:
If `GEF_DEAL_VERSION=1`, the user creates a deal & then clones it. The deal version will be 1 for both.

Cloning is a functionality for makers, deal versioning behaves like a feature flag but keeping compatibility with existing in-flight deals